### DISCUSSION TITLE
License for commercial use

### DISCUSSION THREAD

**Discussion by arihantdaga** [HomeSpan/HomeSpan#51] [Q&A]
Thank you very much @HomeSpan for your great work on this project. It's really great. 
I wanted to use this in one of our commercial project and use this in bridge mode, do i have to obtain an MFI Licence? Or it's not required. I am very happy to attribute this library. But learned that its relatively difficult to obtain a license from apple. 
Can someone help me with this.

**Comment by HomeSpan** [comment]
I'm glad you are finding the library useful but it has been designed using [HAP-R2](https://developer.apple.com/support/homekit-accessory-protocol/), which Apple states it is for non-commercial use only.  I'm unfamiliar with the MFi licensing process or what it entails, but in looking through some of the [MFi Links](https://mfi.apple.com/en/how-it-works.html) it seems that Apple has a list of MFi manufacturers that are licensed to create Accessories on behalf of others.
